### PR TITLE
[Feat] Card component

### PR DESCRIPTION
**Pull request by rushikesh3299**
Card with Badge
Card with shadow
Card with text overlay
card with dismiss

correction in z-index for slider & nav

### CODE REVIEW

File: Components/Card/card.css [hunk 1]
```diff
@@ -1,2 +1,217 @@
...
+  transform:rotate(45deg);
+  top:1.8rem;
+  right:-0.7rem;
+  z-index: -1;
+}
+
+.badge-card::after{
+  position:absolute;
+  content:"Trending";
+  color:white;
```
Comment (ajit-sharmaGH) [inline Components/Card/card.css]: you can use color variables  !

File: Components/Card/card.css [hunk 2]
```diff
@@ -1,2 +1,217 @@
...
+
+/*Card with Horizontal card*/
+.hz-card{
+    width:310px;
+  }
+  
+.hz-card .card-container{
+    width:100%;
+    border:1px solid black;
+    box-shadow: 2px 4px #888888;
```
Comment (ajit-sharmaGH) [inline Components/Card/card.css]: here as well

**Review by ajit-sharmaGH — APPROVED** [review]
everything is looking fine, clean code , commented !

**Review by tusharkandpal — APPROVED** [review]
Looks good! Can use variables for colors if you want.

File: Components/Card/card.css (hunk 1 above)
Comment (rushikesh3299) [reply Components/Card/card.css]: Thanks for suggestion.
I have implemented it.

**Pull request merged**